src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -357,6 +357,23 @@ private function validateAndDenormalize(string $currentClass, string $attribute,
         throw new NotNormalizableValueException(sprintf('The type of the "%s" attribute for class "%s" must be one of "%s" ("%s" given).', $attribute, $currentClass, implode('", "', array_keys($expectedTypes)), gettype($data)));
     }
 
+    /**
+     * Get the types for the given class and attribute.
+     *
+     * @param string $currentClass
+     * @param string $attribute
```

Comment:
should be removed, we don't duplicate the signature when not adding more than what it already tells.

Reply:
Yeah, not sure why I've done that 😄